org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -71,6 +71,22 @@
 		</location>
 		<location includeDependencyDepth="infinite" includeDependencyScopes="compile,provided,runtime" includeSource="true" missingManifest="generate" type="Maven">
 			<dependencies>
+				<dependency>
+				    <groupId>com.sun.xml.bind</groupId>
```

Comment:
Would be better to use one from `https://download.eclipse.org/tools/orbit/downloads/drops/R20221123021534`

Reply:
Hmm, when I moved it to orbit, the build breaks with:
`Error:  Cannot resolve project dependencies:
Error:    Software being installed: org.eclipse.mylyn.commons.feature.group 3.26.0.qualifier
Error:    Missing requirement: org.eclipse.mylyn.commons.ui 3.26.0.qualifier requires 'osgi.bundle; org.apache.commons.lang [2.6.0,3.0.0)' but it could not be found
Error:    Cannot satisfy dependency: org.eclipse.mylyn.commons.feature.group 3.26.0.qualifier depends on: org.eclipse.equinox.p2.iu; org.eclipse.mylyn.commons.ui 0.0.0`

Reply:
Never mind. It's only the source that's available from orbit. Sticking to the maven version for now

Reply:
Orbit searches are weird. I did find the right one, but it has the same problem.
I also tried the 2.3.0 version, but it has similar, but different, problems